docs/data-ai/data/data-pipelines.md

```diff
@@ -0,0 +1,756 @@
+)
+
+data_client.enable_data_pipeline(id="<pipeline-id>")
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+Use [`DataClient.EnableDataPipeline`](https://pkg.go.dev/go.viam.com/rdk/app#DataClient.EnableDataPipeline) to enable a disabled data pipeline:
+
+```go
+client, err := utils.NewViamClient(context.Background(), utils.WithAPIKey("<api-key>", "<api-key-id>"))
+if err != nil {
+    panic(err)
+}
+defer client.Close()
+
+dataClient := client.DataClient()
+
+_, err := dataClient.EnableDataPipeline(context.Background(), &datapb.EnableDataPipelineRequest{
+    Id: "<pipeline-id>",
+})
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+Use [`dataClient.EnableDataPipeline`](/dev/reference/apis/data-client/#enabledatapipeline) to enable a disabled data pipeline:
+
+```typescript
+const apiKey = '<api-key>';
+const apiKeyID = '<api-key-id>';
+
+const client = await createViamClient({
+  credential: {
+    type: 'api-key',
+    payload: { key: apiKey, keyId: apiKeyID }
+  }
+});
+
+const dataClient = client.dataClient;
+
+const pipeline = [
+
+await dataClient.enableDataPipeline({ id: "<pipeline-id>" });
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Disable a pipeline
+
+Disabling a data pipeline lets you pause data pipeline execution without fully deleting the pipeline configuration from your organization.
+
+{{< tabs >}}
+{{% tab name="CLI" %}}
+
+Use [`datapipelines disable`](/dev/tools/cli/#datapipelines) to disable a data pipeline:
+
+```sh {class="command-line" data-prompt="$"}
+viam datapipelines disable --id=<pipeline-id>
+```
+
+{{% /tab %}}
+{{% tab name="Python" %}}
+
+Use [`DataClient.DisableDataPipeline`](/dev/reference/apis/data-client/#disabledatapipeline) to disable a data pipeline:
+
+```python
+data_client = DataClient.from_api_key(
+    api_key="<api-key>",
+    api_key_id="<api-key-id>"
+)
+
+data_client.disable_data_pipeline(id="<pipeline-id>")
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+Use [`DataClient.DisableDataPipeline`](https://pkg.go.dev/go.viam.com/rdk/app#DataClient.DisableDataPipeline) to disable a data pipeline:
+
+```go
+client, err := utils.NewViamClient(context.Background(), utils.WithAPIKey("<api-key>", "<api-key-id>"))
+if err != nil {
+    panic(err)
+}
+defer client.Close()
+
+dataClient := client.DataClient()
+
+_, err := dataClient.DisableDataPipeline(context.Background(), &datapb.DisableDataPipelineRequest{
+    Id: "<pipeline-id>",
+})
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+Use [`dataClient.DisableDataPipeline`](/dev/reference/apis/data-client/#disabledatapipeline) to disable a data pipeline:
+
+```typescript
+const apiKey = "<api-key>";
+const apiKeyID = "<api-key-id>";
+
+const client = await createViamClient({
+  credential: {
+    type: "api-key",
+    payload: { key: apiKey, keyId: apiKeyID },
+  },
+});
+
+const dataClient = client.dataClient;
+
+await dataClient.disableDataPipeline({ id: "<pipeline-id>" });
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Delete a pipeline
```

Comment:
Deleting a pipeline will also delete its run history and the pipeline results collection